acceptance/router_benchmark/brload/main.go

```diff
@@ -0,0 +1,326 @@
+// Copyright 2023 SCION Association
```

Comment:
Maybe in future (not as part of this PR) we could consider a more compact (and machine readable) license header: https://spdx.dev/learn/handling-license-info/
E.g. one liner: `/* SPDX-License-Identifier: Apache-2.0 */`